src/components/AllToDos.tsx

```diff
@@ -0,0 +1,23 @@
+import SingleToDo from "./SingleToDo";
+
+type propsType= {
+    data:{
+        todos:{id: number, title: string; completed: boolean }[],
+        isLoading: boolean,
+        error: string
+     },
```

Comment:
you should split the types furthermore, for readability and clarity

`
type Todo = {id: number, title: string; completed: boolean }

type Data = { 
        todos:Todo[],
        isLoading: boolean,
        error: string
}

type Props = {
data:Data,
onCheckClick(id: number): void;
onDeleteClick(id: number): void;
}
`